Library/Homebrew/compat/cask/dsl.rb

```diff
@@ -5,12 +5,8 @@ def gpg(*)
         odeprecated "the `gpg` stanza", disable_on: Time.new(2018, 12, 31)
       end
 
-      def license(*)
-        odisabled "the `license` stanza"
```

Comment:
I think we shouldn't remove license and cask header token until we have some sort of fix (e.g. https://github.com/Homebrew/brew/pull/4730) for the errors we might get from these.

Reply:
@commitay Good point, thanks.

Reply:
@commitay Added a comment 👍

Reply:
👍 Thanks!